packages/kit/test/apps/basics/src/routes/cookies/+page.svelte

```diff
@@ -3,4 +3,4 @@
 	export let data;
 </script>
 
-Cookie: <span id="cookie-value">{data.cookie}</span>
+Cookie: <span id="cookie-value">{data.cookie ?? 'undefined'}</span>
```

Comment:
is this `?? 'undefined'` pattern precise enough? the assertion tests for undefined because thats what svelte4 output added there, with this change, it becomes a little less precise as null would also be rendered as 'undefined'.

The other option would be to import the svelte version into the testcase and assert empty string for svelte5 and undefined for svelte4

Reply:
How about printing the corresponding value such as in Svelte 4:
```js
`${data.cookie}` // prints null or undefined 
```